src/sentry/models/identity.py

```diff
@@ -15,10 +17,8 @@
 
 logger = logging.getLogger(__name__)
 
-# TODO(dcramer): pull in enum library
 
-
-class IdentityStatus:
+class IdentityStatus(Enum):
```

Comment:
I'd recommend doing this in a separate PR. This is used in a lot of places throughout the codebase and is breaking your tests currently.

Reply:
👍 Can we merge this in separately?